.erda/migrations/core-services/20220713-erda_rule_set-base.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS `erda_rule_set` (
```

Comment:
`rule set` or just `rule`, I think each record in table is a rule.

Reply:
rule is used.